toxic_timeout.go

```diff
@@ -8,6 +11,10 @@ type TimeoutToxic struct {
 	Enabled bool `json:"enabled"`
 	// Times in milliseconds
 	Timeout int64 `json:"timeout"`
+	// If true, use Toxicity
+	SometimesToxic bool `json:"sometimesToxic"`
+	// 'Toxicity' or probability of timing out 0..1
+	Toxicity float32 `json:"toxicity"`
```

Comment:
It's a bit of a hack, but it's less hacky than 2 fields I guess. (No way to have a 0 toxicity toxic, but why would you want that anyway?)